model_compression_toolkit/exporter/model_exporter/keras/fakely_quant_keras_exporter.py

```diff
@@ -121,6 +121,10 @@ def _unwrap_quantize_wrapper(layer: Layer):
 
             return layer
 
+        # Delete metadata layer is exists
+        if hasattr(self.model, 'metadata_layer'):
+            delattr(self.model, 'metadata_layer')
```

Comment:
Maybe add a warning that metadata is not exported in the TFLITE case? (all tflite cases where you delete the metadat_layer)